From 2adc906c5433403562c4d58105e000aee4360bcf Mon Sep 17 00:00:00 2001
From: imboyce <imboyce@users.noreply.github.com>
Date: Tue, 27 Dec 2016 15:15:37 +0800
Subject: [PATCH] Initial commit

---
 README.md | 2 ++
 1 file changed, 2 insertions(+)
 create mode 100644 README.md

diff --git a/README.md b/README.md
new file mode 100644
index 0000000..74d8510
--- /dev/null
+++ b/README.md
@@ -0,0 +1,2 @@
+# LearnIt
+How to use it
-- 
1.9.5.msysgit.0

